apps/vs-code-designer/src/app/commands/initProjectForVSCode/InitVSCodeStepBase.ts

```diff
@@ -114,53 +113,56 @@ export abstract class InitVSCodeStepBase extends AzureWizardExecuteStep<IProject
   }
 
   private async writeTasksJson(context: IProjectWizardContext, vscodePath: string): Promise<void> {
```

Comment:
Same thing as for the func host start function. It would be great to have a different function